docs/developer-guide/setup.md

```diff
@@ -56,11 +62,11 @@ make dep
 **Congratulations!** You can now build the frontend assets (JS), compile the backend binary (Go) and then run a custom PhotoPrism version in your local environment:
 
 ```bash
-make build-js
-make build-go
+make build-js && make build-go
```

Comment:
```suggestion
make build-js build-go
```

It should have the same behaviour without repeating `make`.

Reply:
True. Note, though, that the idea of listing the commands separately was to show that you can build frontend and backend independently. If you always run `make build-js build-go`, it takes much longer and I've found that not everyone understands that...

Reply:
Agreed. I'll change it with some explaination about the two options.